### ISSUE TITLE
Cannot find module 'ionic-context-menu'

### ISSUE THREAD

**Issue by yurymkomarov** [smartin85/ionic-context-menu#1]
After installation module cannot be find. There is no index file in module package in node_modules folder

```
komarov:mobile yuri$ tree node_modules/ionic-context-menu/
node_modules/ionic-context-menu/
├── LICENSE
├── README.md
├── karma.conf.js
├── ng-package.json
├── ng-package.prod.json
├── package.json
├── src
│   ├── lib
│   │   ├── ionic-context-menu-content.component.ts
│   │   ├── ionic-context-menu-item.component.ts
│   │   ├── ionic-context-menu.component.spec.ts
│   │   ├── ionic-context-menu.component.ts
│   │   └── ionic-context-menu.module.ts
│   ├── public_api.ts
│   └── test.ts
├── tsconfig.lib.json
├── tsconfig.spec.json
└── tslint.json

2 directories, 16 files
```

----

Doesn't work:

```
import { IonicContextMenuModule } from 'ionic-context-menu';
```

Does work:
```
import { IonicContextMenuModule } from '../../node_modules/ionic-context-menu/src/lib/ionic-context-menu.module';
```

**Comment by hrecho** [comment]
same issue here,  
for me even the relative path is not working,

also  i tried
`import { IonicContextMenuModule } from 'ionic-context-menu/src/lib/ionic-context-menu.module';`

i am getting
`
ERROR in ./node_modules/ionic-context-menu/src/lib/ionic-context-menu.module.ts
Module build failed (from ./node_modules/@ngtools/webpack/src/index.js):
Error: \node_modules\ionic-context-menu\src\lib\ionic-context-menu.module.ts is missing from the TypeScript compilation. Please make sure it is in your tsconfig via the 'files' or 'include' property.
The missing file seems to be part of a third party library. TS files in published libraries are often a sign of a badly packaged library. Please open an issue in the library repository to alert its author and ask them to package the library using the Angular Package Format (https://goo.gl/jB3GVv).
    at AngularCompilerPlugin.getCompiledFile (\node_modules\@ngtools\webpack\src\angular_compiler_plugin.js:752:23)
    at plugin.done.then (\node_modules\@ngtools\webpack\src\loader.js:41:31)
    at process._tickCallback (internal/process/next_tick.js:68:7)
`

**Comment by smartin85** [comment]
I´m so sorry, currently I dont´t have time to fix this. Feel free to send a pr.
Meanwhile you could try to downgrade to an earlier version:
> npm install --save ionic-context-menu@1.0.0

**Comment by yurymkomarov** [comment]
@smartin85 , roger that! Will try to do it ASAP

**Comment by 0afcode** [comment]
I'm also having the same issues as the others posted above. Did anyone find a solution that they can share?

**Comment by kytwb** [comment]
@0afcode `npm install --save ionic-context-menu@1.0.0` works for now.

**Comment by smartin85** [comment]
Should be fixed in 1.0.3
